### PR TITLE
Add github plugins installation guide

### PR DESCRIPTION
**Pull request by ngoedde**

### CODE REVIEW

File: source/sysadmins-guide/github-plugin-guide/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: Shopware 5 GitHub plugin guide
+github_link: sysadmins-guide/github-plugin-guide/index.md
+indexed: true
+---
+## Introduction
+This guide teaches you how to install all the Shopware plugins that can be found under <a href="https://github.com/shopwareLabs">GitHub - shopwareLabs</a>.
```
Comment (mitelg) [inline source/sysadmins-guide/github-plugin-guide/index.md]: ... install the Shopware...

File: source/sysadmins-guide/github-plugin-guide/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Your directory structure should now be something like this:
+**Frontend/SwagPaymentPaypal/**
+
+### Step 3 - Re-creating the archive
+Now create a new .zip archive with Frontend as the root directory in it.
+It should then look like this:
+**SwagPaymentPaypal.zip/Frontend/SwagPaymentPaypal/**
+
+### Step 4 - Uploading the plugin
+The archive we have creates will be supported for remote upload in the plugin manager.
```
Comment (mitelg) [inline source/sysadmins-guide/github-plugin-guide/index.md]: ... we have created ...

**Comment by bcremer** [comment]
The guide seems overly complex to me.
As an alternativ we should consider adding installable release zip files to the repository like @janbuecker did here for https://github.com/shopwareLabs/SwagMediaS3 with this commit: https://github.com/shopwareLabs/SwagMediaS3/commit/9f63591ef78a5daddfe353b5c61170b0c414feb9

This way we would also use proper tags and don't teach people to install from the master branch.

So please don't merge this PR yet.

**Comment by mitelg** [comment]
I think there is no need for that

**Pull request closed without merging**